table/server/common/src/test/java/alluxio/table/common/udb/PathTranslatorTest.java

```diff
@@ -76,24 +76,24 @@ public void multipleSubdir() throws Exception {
 
   @Test
   public void samePathDifferentUfs() throws Exception {
-    String alluxioPath  = "alluxio:///my/table/directory";
+    String alluxioPath  = "alluxio://master/my/table/directory";
```

Comment:
we should also be able to support path like 'alluxio:///dir/file' does this change not work with that kind of path?

Reply:
https://github.com/Alluxio/alluxio/blob/90b3c7b010f926be3c2df7519eec067df30fd2a7/table/server/common/src/main/java/alluxio/table/common/udb/PathTranslator.java#L96-100

I check for the existence of both scheme and authority. Is this too strict?

Reply:
I think it is ok to not have authority, specifically i think for HA URIs. see https://docs.alluxio.io/os/user/stable/en/deploy/Running-Alluxio-On-a-HA-Cluster.html#specify-alluxio-service-in-configuration-parameters-or-java-options

Reply:
Changed in 825c0cc.